Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM debian:jessie
```

Comment:
Yes libmusl has some drawbacks, have you tried with alpine:edge which uses a more recent release of musl? Anyway if you have some debug infos should je great to report the build error to musl.